### DISCUSSION TITLE
Toolbar in Taskbar - possible

### DISCUSSION THREAD

**Discussion by t8769** [valinet/ExplorerPatcher#227] [Q&A] [answered]
Downloaded Portable Start menu but can't find it anywhere in win11.

How can I get back Toolbars in the Taskbar?

Is this possible with this software?

Where can I find and start it?

Thank you

**Comment by valinet** [comment, marked as answer]
Specifically about this software, reading the [README](https://github.com/valinet/ExplorerPatcher/blob/master/README.md) is a very very very good starting point. EP enables the Windows 10 taskbar, with all the features that had.

Thanks

**Comment by t8769** [comment]
Ok thanks
